src/main/kotlin/org/openmbee/flexo/mms/routes/ldp/LockRead.kt

```diff
@@ -0,0 +1,130 @@
+package org.openmbee.flexo.mms.routes.ldp
+
+import io.ktor.http.*
+import io.ktor.server.response.*
+import org.openmbee.flexo.mms.*
+import org.openmbee.flexo.mms.server.LdpDcLayer1Context
+import org.openmbee.flexo.mms.server.LdpGetResponse
+import org.openmbee.flexo.mms.server.LdpHeadResponse
+
+private val SPARQL_BGP_LOCK = """
+    graph mor-graph:Metadata {
+        ?_lock a mms:Lock ;
+            mms:etag ?__mms_etag ;
+            ?lock_p ?lock_o .
+        
+        optional {
+            ?thing mms:lock ?_lock ;
+                ?thing_p ?thing_o .
+        }
+    }
+    
+    ${permittedActionSparqlBgp(Permission.READ_LOCK, Scope.LOCK)}
+"""
+
+private val SPARQL_SELECT_LOCK = """
+    select distinct ?__mms_etag {
+        $SPARQL_BGP_LOCK
+    } order by asc(?__mms_etag)
+"""
+
+private val SPARQL_CONSTRUCT_LOCK = """
+    construct {
+        ?_lock ?lock_p ?lock_o ;
+            mms:etag ?__mms_etag .
+        
+        ?thing ?thing_p ?thing_o .
+        
+        ?_context a mms:Context ;
+            mms:permit mms-object:Permission.ReadLock ;
+            mms:policy ?policy .
+        
+        ?__mms_policy ?__mms_policy_p ?__mms_policy_o .
```

Comment:
this is returning the entire policies graph

Reply:
`?_context` is supposed to be applying whichever scoped policy is relevant to reading from the resource (the Lock in this case), but need to verify if that variable is binding from `permittedActionSparqlBgp` in AccessControl.kt

Reply:
the same thing also happens in BranchRead